tests/model_serving/maas_billing/multitenancy/aigateway/conftest.py

```
@@ -0,0 +1,49 @@
+    return AIGATEWAY_INFRA_NAMESPACE
+
+
+@pytest.fixture
+def aigateway_for_test(
+    admin_client: DynamicClient,
+    aigateway_infra_namespace: str,
+    teardown_resources: bool,
+) -> Generator[AIGatewayTestContext, Any, Any]:
+    """Create a disposable AIGateway and yield context for bootstrap assertions."""
+    aigateway_name = f"e2e-aigw-{generate_random_name()}"
+    tenant_namespace_name = tenant_namespace_name_for_aigateway(aigateway_name=aigateway_name)
```

Flake8 (7.3.0)

[error] local variable 'tenant_namespace_name' is assigned to but never used

(F841)